app/controllers/users/verify_personal_key_controller.rb

```diff
@@ -30,6 +31,10 @@ def create
           **result.to_h,
           pii_like_keypaths: [[:errors, :personal_key], [:error_details, :personal_key]],
         )
+        irs_attempts_api_tracker.personal_key_reactivation_submitted(
+          success: result.success?,
+          failure_reason: result.errors,
```

Comment:
I think we do `result.error_details` right?

Reply:
based on the verify_form form response returns `FormResponse.new(success: valid?, errors: errors, extra: extra)`